docs/content/getting-started.md

```diff
@@ -1,481 +1,300 @@
 
-![OAuth Tokens](/img/boltpy/bot-token.png "Bot OAuth Token")
+:::
 
-5. Then head over to **Basic Information** and scroll down under the App Token section and click **Generate Token and Scopes** to generate an app-level token. Add the `connections:write` scope to this token and save the generated `xapp` token, we'll use both these tokens in just a moment. 
+## Creating a project {#creating-a-project}
 
-6. Navigate to **Socket Mode** on the left side menu and toggle to enable. 
+With the toolchain configured, it's time to set up a new Bolt project. This contains the code that handles logic for your app.
 
-:::tip  
+If you don’t already have a project, let’s create a new one!
 
-Treat your tokens like passwords and [keep them safe](https://docs.slack.dev/authentication/best-practices-for-security). Your app uses tokens to post and retrieve information from Slack workspaces.
+<Tabs groupId="cli-or-terminal">
+<TabItem value="cli" label="Slack CLI">
 
-:::
+A starter template can be used to start with project scaffolding:
 
----
+```sh
+$ slack create first-bolt-app --template slack-samples/bolt-python-getting-started-app
+$ cd first-bolt-app
+```
 
-### Setting up your project {#setting-up-your-project}
+After a project is created you'll have a `requirements.txt` file for app dependencies and a `.slack` directory for Slack CLI configuration.
 
-With the initial configuration handled, it's time to set up a new Bolt project. This is where you'll write the code that handles the logic for your app.
+A few other files exist too, but we'll visit these later.
 
-If you don’t already have a project, let’s create a new one. Create an empty directory:
+</TabItem>
+<TabItem value="terminal" label="Terminal">
 
-```shell
-mkdir first-bolt-app
-cd first-bolt-app
+A starter template can be cloned to start with project scaffolding:
+
+```sh
+$ git clone https://github.com/slack-samples/bolt-python-getting-started-app first-bolt-app
+$ cd first-bolt-app
+$ npm install
 ```
 
-Next, we recommend using a [Python virtual environment](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment) to manage your project's dependencies. This is a great way to prevent conflicts with your system's Python packages. Let's create and activate a new virtual environment with [Python 3.6 or later](https://www.python.org/downloads/):
+Outlines of a project are taking shape, so we can move on to running the app!
 
-```shell
+</TabItem>
+</Tabs>
+
+We recommend using a [Python virtual environment](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment) to manage your project's dependencies. This is a great way to prevent conflicts with your system's Python packages. Let's create and activate a new virtual environment with [Python 3.6 or later](https://www.python.org/downloads/):
+
+```sh
 python3 -m venv .venv
 source .venv/bin/activate
 ```
```

Comment:
````suggestion
```sh
python3 -m venv .venv
source .venv/bin/activate
pip install -r requirements.txt
```
````

👁️‍🗨️ We might soon want to revisit this, but for now this might be a requirement.

Reply:
🔍 Also I'm curious if prefixing commands with `$` should be avoided? I notice I'm often inconsistent and include this but am open to either!

Reply:
👾 The command to `install` might also follow the check for a virtual environment if we want to guard against global installs-